AWS Cloudformation proposal

stvhawes opened the pull request.

stvhawes commented:
Hi, I hope this meets contributing standards; feel free to edit.

overint commented on aws/paypal-ipn-cloudformation.yml:
```diff
@@ -0,0 +1,352 @@
+    Description: PaypalMessageVerificationURL
+    Type: String
+    Default: https://ipnpb.sandbox.paypal.com/cgi-bin/webscr   # for Sandbox IPNs
+    #    Default: https://ipnpb.paypal.com/cgi-bin/webscr   # for live IPNs
```
Maybe add a comment to the readme about switching this?

stvhawes replied:
done.  added a section describing each parameter.

overint commented on aws/paypal-ipn-cloudformation.yml:
```diff
@@ -0,0 +1,352 @@
+            - '    print("VERIFIED")'
+            - !Join 
+              - ''
+              - - '    response = sqs.send_message(QueueUrl="https://sqs.ap-southeast-2.amazonaws.com/127569160727/'
```
Could we maybe switch these urls to `<change me>` or similar?

stvhawes replied:
parameterised them, thanks for the pickup.

overint commented on aws/paypal-ipn-cloudformation.yml:
```diff
@@ -0,0 +1,352 @@
+          text/html: Empty
+
+# Stages
+  # Invoke URL: https://i681ypmtg2.execute-api.ap-southeast-2.amazonaws.com/dev
```
As above.

stvhawes replied:
as above.

stvhawes commented:
thanks overint!  I'll fix those items + retest

stvhawes commented:
some other changes added + happy path tested.
* tightened security policies sending to SNS and SQS
* added the throttles from the README and updated the readme with more throttle detail

overint commented:
Thanks for making those changes!
